tests/framework_flask/tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+setupdir = {toxinidir}/../..
+envlist =
+    {pypy,py27,py35,pypy3}-flask0012,
```

Comment:
Question: should we include `pypy3` in this test line? `pypy3` is 3.6+ compatible at this point in time, and moving to py37 compatibility. If flask only supports py35 at this version, wouldn't `pypy3` potentially cause issues?

Reply:
Potentially, I'm ok with removing it. It's a very old version of flask anyway, so unlikely to have issues related to it.